plugins/pixart-rf/fu-pxi-device.c

```diff
@@ -599,10 +615,22 @@ static gboolean
 fu_pxi_device_setup (FuDevice *device, GError **error)
 {
 	FuPxiDevice *self = FU_PXI_DEVICE (device);
+	g_autofree gchar *devid = NULL;
+	struct hidraw_devinfo	hid_raw_info;
+	if (!fu_pxi_device_get_raw_info (self, &hid_raw_info ,error))
+		return FALSE;
 	if (!fu_pxi_device_fw_ota_init (self, error))
 		return FALSE;
 	if (!fu_pxi_device_fw_get_info (self, error))
 		return FALSE;
+
+	devid = g_strdup_printf ("HIDRAW\\VID_%04hX&PID_%04hX&BLE_DEV_%s",
+				hid_raw_info.vendor,
+				hid_raw_info.product,
+				fu_device_get_name (device));
+				
+	fu_device_set_logical_id (device, devid);
```

Comment:
The logical ID is supposed to tell fwupd how to tell apart the devices if you have two devices of the same type plugged in. I'd expect something like a bluetooth connection ID rather than a name here.

Reply:
We have to add the model name command for different devices in next version . This default device name is fine for us.